templates/_helpers.tpl

```diff
@@ -209,7 +209,7 @@ Create the name of the stackstorm-ha service account to use
     - |
       /bin/cp -aR /opt/stackstorm/packs/. /opt/stackstorm/packs-shared &&
       /bin/cp -aR /opt/stackstorm/virtualenvs/. /opt/stackstorm/virtualenvs-shared
-  {{- with .Values.securityContext }}
+  {{- with $.Values.securityContext }}
```

Comment:
`.` here refers to the an element of `.Values.st2.packs.images` and therefore does not have a `securityContext`. This change ensures that we use the global `.Values.securityContext`. Please see [this line](https://github.com/StackStorm/stackstorm-ha/blob/fe9b3a5ae93a1dc8002e39ce8a51686275dfba87/templates/_helpers.tpl#L197) for more details.